site/content/en/docs/Tutorials/allocator-service-go.md

```diff
@@ -4,7 +4,12 @@ linkTitle: "Create an Allocator Service (Go)"
 date: 2019-01-03T03:15:34Z
 description: >
   This tutorial describes how to interact programmatically with the 
+{{% feature expiryVersion="0.12.0" %}}
```

Comment:
It looks like I can't use the curly braces in the yaml part here, since it causes the tests to fail. So I'm going to leave this with the v1alpha1 version and we'll need to change it around the time we cut the 0.12.0 release.